src/main/java/org/mule/tools/apikit/model/MuleConfig.java

```diff
@@ -20,28 +21,53 @@
 import java.util.Objects;
 
 import static java.util.stream.Collectors.toList;
+import static org.apache.commons.collections.CollectionUtils.*;
```

Comment:
Please, avoid wildcard imports. Remove the one that is unused and here import `isNotEmpty` only.